constructor/nsis/_nsis.py

```diff
@@ -7,6 +7,7 @@
 # directory is merely a convenience.  This way, functionally can easily
 # be tested in an installation.
 
+import logging
```

Comment:
I wouldn't touch this file either. There are a few moving pieces around the stdout / stderr in this part of the logic (e.g. capturing the streams from ctypes and whatnot), and even if it _should_ work, who knows what's going to break.